packages/atlas/src/components/_video/VideoHero/VideoHero.styles.ts

```diff
@@ -92,3 +93,7 @@ export const ButtonsContainer = styled.div`
     flex-shrink: 0;
   }
 `
+
+export const TitleText = styled(Text)`
+  overflow-y: auto;
```

Comment:
I think it should depend on `isCategory` prop, now it will be applied also to category video hero

Reply:
Thanks, I applied this.